frontend_lib/src/tracimLiveMessage.js

```diff
@@ -1,6 +1,14 @@
+import { CONTENT_TYPE } from './helper.js'
+
 // TLM Event Type list
 const USER = 'user'
-const CONTENT = 'content'
+const CONTENT = {
```

Comment:
as from the latest spec, the content type must be put after the core event type.
I think you might be putting the content type in between the event type and the core event type here

Reply:
Changed.